lib/measures/tbd/measure.rb

```diff
@@ -1,9 +1,4 @@
-begin
-  # Try to load from the tbd gem.
-  require "tbd"
-rescue LoadError
-  require_relative "resources/tbd"
-end
+require_relative "tbd"
```

Comment:
Assembling the measure itself from static TBD files - no point looking up a gem.